docker-compose.yml

```
@@ -23,7 +23,7 @@ services:
     container_name: territory_api
     command: --start-with-debugging
     environment:
```

Perhaps we could define an environment file with [env_file](https://docs.docker.com/compose/compose-file/#env_file) in Docker Compose, something like the following?

```suggestion
    env_file: development.env
```
(We'd have to ditch the list below too... Need support for multi-line suggestions 😒)

That way we could just add `*.env` to the `.gitignore` to remove any chance of committing values.

We could also add a `whatever.env.sample` as well with all required and optional environment variables defined, which we could copy if there is no existing `*.env` file on a pre-install hook if we wanted.

It _could_ remove the need for a dependency on `dotenv` too, if we were to assume that the application would _always_ be run in Docker (i.e. with the environment already defined).